### DISCUSSION TITLE
Providing IP addresses instead of local host ip

### DISCUSSION THREAD

**Discussion by madhugit2022** [c-jimenez/open-ocpp#122] [General]
Hi all,

I tried changing the ConnexionUrl to an IP address instead of the default localhost ip. I am unable to connect to central system. Can anyone say what is the issue? 

Tried both the formats. No luck

ConnexionUrl=wss://192.168.1.20/openocpp/

ConnexionUrl=wss://192.168.1.20:8080/openocpp/

**Comment by c-jimenez** [comment]
Hello,

Can you provide your configuration file and your logs output?

Have you got a system wide proxy enabled through the environment vars http_proxy and https_proxy? If yes, you may have to remove them to allow connection to IP addresses that are is the no_proxy list.

Also this is a secured connection you're trying to establish (wss) to a local IP address. Does your central system have a non self-signed certificate matching the given IP address?
To check if your connection troubles are linked to HTTPS misconfiguration you can change the following vars in configuration file to  true (not recommended in production) :
```
TlsAllowSelfSignedCertificates=true
TlsAllowExpiredCertificates=true
TlsAcceptNonTrustedCertificates=true
TlsSkipServerNameCheck=true
```
If this works, you may disable each of these configuration values one at a time to figure out the root cause of your issue.
